app/Phragile/AssigneeCollection.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Phragile;
+
+class AssigneeCollection {
+
+	private $assignees;
+
+	public function __construct(PhabricatorAPI $phabricator, array $tasks)
+	{
+		$assigneePHIDs = $this->extractUniqueAssignees($tasks);
+		$this->assignees = $this->processAssignees(
+			$phabricator->getUserData(['phids' => $assigneePHIDs])
+		);
```

Comment:
Actually, this is not used as a collection at all, so perhaps it does not matter it is not implemented as such. If your only public method is `getName($phid)`, then this to the outside looks more like a `AssigneeRepository`.